pkg/identity/numericidentity.go

```diff
@@ -168,12 +168,15 @@ func InitWellKnownIdentities(c Configuration) int {
 	//   k8s:io.kubernetes.pod.namespace=<NAMESPACE>
 	//   k8s:io.cilium/app=etcd-operator
 	//   k8s:io.cilium.k8s.policy.cluster=default
-	WellKnown.add(ReservedETCDOperator, []string{
+	etcdOperatorLabels := []string{
 		"k8s:io.cilium/app=etcd-operator",
 		fmt.Sprintf("k8s:%s=%s", api.PodNamespaceLabel, c.CiliumNamespaceName()),
 		fmt.Sprintf("k8s:%s=cilium-etcd-sa", api.PolicyLabelServiceAccount),
 		fmt.Sprintf("k8s:%s=%s", api.PolicyLabelCluster, c.LocalClusterName()),
-	})
+	}
+	WellKnown.add(ReservedETCDOperator, etcdOperatorLabels)
+	WellKnown.add(ReservedETCDOperator, append(etcdOperatorLabels,
```

Comment:
This will not have the behavior that we want, `WellKnown` has a map of identities that maps to labels, this means the second `WellKnown.add` will replace the previous one. I think we will need one different identity for each existing well known set of labels.

Reply:
Oh right. I tried to explore different ideas to avoid having two well-known identities for each component but it's just easier to have both of them. 
I reorganized the code a bit to avoid having to write all the numbers, I think we can change the numbers for the already existing ones, can't we?

Reply:
>I reorganized the code a bit to avoid having to write all the numbers, I think we can change the numbers for the already existing ones, can't we?

we can but we should avoid doing it for existing ones.

Reply:
Right. I updated the code to keep the same value for the existing ones. I kept the change to avoid having to write the value for each of them. If you think it doesn't make sense I can just undo it and write by hand the value of all of them...